### ISSUE TITLE
Add Google Analytics tags

### ISSUE THREAD

**Issue by gpazuch** [ns1labs/orb#209]
Add Google Analytics tags and create another build config and environment for beta.orb.live --prod build

**Comment by gpazuch** [comment]
https://angular.io/cli/usage-analytics-gathering

https://angular.io/cli/analytics

https://github.com/ns1labs/orb/blob/fba26894988e2561db7afba7dd136b09de6459ef/ui/angular.json#L186

**Comment by gpazuch** [comment]
On implementing Google Analytics I found [this code](https://github.com/ns1labs/orb/blob/develop/ui/src/app/%40core/utils/analytics.service.ts) that could come in handy.
It seems to use an ID tag from `angular.json` `cli:{ analytics: {id: "ID_HASH"}` 
I'll have to investigate this approach further.

we can probably assign the GTM provided function to `ga['send']`
as soon as the script is loaded up, just like Google instructs, but with the `defer` attribute assigned to the respective `<src>`  

```html
<script defer scr="googleGTM...">
```  

For now I have installed [angular-google-tag-manager](https://www.npmjs.com/package/angular-google-tag-manager) the app, which should work very similarly to `AnalyticsService` in the current code.

It should allow easier environment set ups for different build targets, and also provides the full features of GTM tracking.
